src/pyOpenMS/pxds/ProteaseDigestion.pxd

```diff
@@ -1,62 +1,58 @@
 from Types cimport *
 from String cimport *
 from AASequence cimport *
+from DigestionEnzyme cimport *
 
 cdef extern from "<OpenMS/CHEMISTRY/ProteaseDigestion.h>" namespace "OpenMS":
 
-    cdef cppclass ProteaseDigestion:
+    cdef cppclass ProteaseDigestion(EnzymaticDigestion):
+        # wrap-inherits:
+        #    DigestionEnzyme
+        #
+        # wrap-doc:
+        #     Class for the enzymatic digestion of proteins
+        #
+        #     Digestion can be performed using simple regular expressions, e.g. [KR] | [^P] for trypsin.
+        #     Also missed cleavages can be modeled, i.e. adjacent peptides are not cleaved
+        #     due to enzyme malfunction/access restrictions. If n missed cleavages are allowed, all possible resulting
+        #     peptides (cleaved and uncleaved) with up to n missed cleavages are returned.
+        #     Thus no random selection of just n specific missed cleavage sites is performed.
 
       ProteaseDigestion() nogil except +
-      ProteaseDigestion(ProteaseDigestion) nogil except + #wrap-ignore
+      ProteaseDigestion(ProteaseDigestion) nogil except +
 
-      SignedSize getMissedCleavages() nogil except +
-      void setMissedCleavages(SignedSize missed_cleavages) nogil except +
-
-      # not wrapped due to name clash with DigestionEnzyme.h (???)
-
-      # void setEnzyme(DigestionEnzyme* enzyme) nogil except +
-
-      String getEnzymeName() nogil except +
-      void setEnzyme(String name) nogil except +
+      void setEnzyme(String name) nogil except + # wrap-doc:Sets the enzyme for the digestion (by name)
 
       Size digest(AASequence & protein, libcpp_vector[AASequence] & output) nogil except +
+
       Size digest(AASequence & protein, libcpp_vector[AASequence] & output, Size min_length, Size max_length) nogil except +
-      Size peptideCount(AASequence & protein) nogil except +
+          # wrap-doc:
```

Comment:
@jpfeuffer Do you think we should use such a format for more complex methods (# wrap-doc)? To explain the parameters and return value individually. I am not sure how that will then be represented in the IDE/help.

Reply:
It seems @rahul799 is doing that already, but with specifying the relevant param via `:param protein:`.
@rahul799 How is that represented in the IDE?

Reply:
Yes, I am using that syntax. I tried checking it in the IDE but as the IntelliSense is not working in vs code, so I am not very sure, how it will look like.